// stdafx.h : 标准系统包含文件的包含文件，
// 或是经常使用但不常更改的
// 特定于项目的包含文件
//

#pragma once

#include "targetver.h"

#define WIN32_LEAN_AND_MEAN             //  从 Windows 头文件中排除极少使用的信息

#include "common.h"
#include "protocol/MessageBase.pb.h"
#include "protocol/QQLoginCaptchaConfirm.pb.h"
#include "protocol/QQLoginCaptchaNotify.pb.h"
#include "protocol/QQLoginReq.pb.h"
#include "protocol/RobotLoginReq.pb.h"
#include "protocol/RobotLoginRsp.pb.h"

// TODO: 在此处引用程序需要的其他头文件
